lib/rules/ruleInstantiations.ts

```diff
@@ -63,8 +63,8 @@ export class RuleInstantiation extends RuleBase implements IRule {
             message: `no ${kind} ${association.formalPart.map(name => name.nameToken.text).join(', ')} on ${typeName}`,
             code
           });
-        } else {
-          foundElements.push(interfaceElement);
+        } else if (association.actualIfInput[0]?.nameToken.getLText() !== 'open') {
+          foundElementsNotOpen.push(interfaceElement);
```

Comment:
This also generates an error for open outputs which are fine imo. A MWE which should be placed in `test_no_error`:
```vhdl
entity dummy_instantiation_open is
  port (
    foo : out integer
    );
end entity;
entity instantiation_open is
end entity;
architecture arch of instantiation_open is
begin
  dummy : entity work.dummy_instantiation_open
    port map (
      foo => open
      );
end architecture;
```